t/plugin/response-rewrite3.t

```diff
@@ -53,6 +53,45 @@ add_block_preprocessor(sub {
             }
         }
     }
+
+    server {
+        listen 11452;
+        location /brotli_hello {
+            content_by_lua_block {
+                ngx.req.read_body()
+                local s = "hello world hello world hello world"
+                ngx.header['Content-Length'] = #s + 1
+                ngx.say(s)
+            }
+            header_filter_by_lua_block {
+                local conf = {
+                    comp_level = 6,
+                    http_version = 1.1,
+                    lgblock = 0,
+                    lgwin = 19,
+                    min_length = 1,
+                    mode = 0,
+                    types = "*",
+                }
+                local brotli = require("apisix.plugins.brotli")
```

Comment:
if I am not wrong, you are importing this because nginx doesn't come with brotli by default.

Reply:
yes, in this part we use the functions from `brotli` plugin.